udata/search/commands.py

```diff
@@ -62,32 +61,40 @@ def index_model(adapter, start, reindex=False, from_datetime=None):
     for indexable, doc in docs:
         try:
             if indexable:
-                action = KafkaMessageType.REINDEX if reindex else KafkaMessageType.INDEX
+                action = EventMessageType.REINDEX if reindex else EventMessageType.INDEX
             elif not indexable and not reindex:
-                action = KafkaMessageType.UNINDEX
+                action = EventMessageType.UNINDEX
             else:
                 continue
             message_type = f'{adapter.model.__name__.lower()}.{action.value}'
-            produce(
-                kafka_uri=current_app.config.get('KAFKA_URI'),
-                topic=get_topic(message_type),
-                service='udata',
-                key_id=doc['id'],
-                document=doc,
-                meta={'message_type': message_type, 'index': index_name}
-            )
+            url = f"{current_app.config['SEARCH_SERVICE_API_URL']}/reindex"
+            try:
+                payload = {
```

Comment:
See my related comment on udata-search-service: https://github.com/opendatateam/udata-search-service/pull/35#discussion_r1034948159.
More generally, I find it weird to call reindex with an unindex message type, and an optional new index_name.